app/serializers/billing-detail.js

```diff
@@ -0,0 +1,8 @@
+import ApplicationSerializer from './application';
+
+export default ApplicationSerializer.extend({
+  extract: function(store, type, payload, id, requestType) {
+    this.__requestType = requestType; // used by `extractMeta`
```

Comment:
@blakepettersson - actually, after reviewing dashboard and API, it looks like you are using `store.find('billingDetail', $id')` rather than trying to serialize a singleton resource.  Is this serializer necessary at all anymore?